[FR] Persian Language

### Description

Hi,
I want to contribute to this project by translating and add Persian language to it, can I do that? and would you like to add this language to the project?

### Impact

Persian people who don't know English can use the app

### Additional Context

_No response_

I hope you are doing well, I have made the translation JSON file for the Persian language. You can check the files in my fork here: https://github.com/AmirLavasani/AppFlowy/tree/feat/persian_lang_2836.

However, I cannot build the project to test the translations. Here is the log for the error. can you check this out?

Here is the command I ran:
`cargo make --profile production-linux-x86_64 appflowy`

The main error line is this:
`[   +1 ms] [+11018 ms] packages/flowy_infra_ui/lib/widget/dialog/styled_dialogs.dart:74:27: Error: The argument type 'BorderRadius?' can't
be assigned to the parameter type 'BorderRadiusGeometry' because 'BorderRadius?' is nullable and 'BorderRadiusGeometry' isn't.`

More of the log:
`[  +67 ms] Building Linux application...
[  +11 ms] executing: [build/linux/x64/release/] cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DFLUTTER_TARGET_PLATFORM=linux-x64
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux
[ +132 ms] -- The CXX compiler identification is Clang 10.0.0
[   +2 ms] -- Check for working CXX compiler: /snap/flutter/current/usr/bin/clang++
[ +109 ms] -- Check for working CXX compiler: /snap/flutter/current/usr/bin/clang++ -- works
[        ] -- Detecting CXX compiler ABI info
[  +79 ms] -- Detecting CXX compiler ABI info - done
[   +8 ms] -- Detecting CXX compile features
[        ] -- Detecting CXX compile features - done
[   +4 ms] -- Found PkgConfig: /snap/flutter/current/usr/bin/pkg-config (found version "0.29.1") 
[        ] -- Checking for module 'gtk+-3.0'
[  +17 ms] --   Found gtk+-3.0, version 3.24.20
[  +60 ms] -- Checking for module 'glib-2.0'
[  +12 ms] --   Found glib-2.0, version 2.64.6
[  +33 ms] -- Checking for module 'gio-2.0'
[  +12 ms] --   Found gio-2.0, version 2.64.6
[  +44 ms] -- Checking for module 'keybinder-3.0'
[  +17 ms] --   Found keybinder-3.0, version 0.3.2
[  +67 ms] -- Configuring done
[  +26 ms] -- Generating done
[        ] -- Build files have been written to: /home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/build/linux/x64/release
[   +8 ms] executing: ninja -C build/linux/x64/release install
[   +8 ms] ninja: Entering directory `build/linux/x64/release'
[+21489 ms] [1/19] Generating
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/libflutter_linux_gtk.so,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_basic_message_channel.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_binary_codec.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_binary_messenger.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_dart_project.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_engine.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_json_message_codec.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_json_method_codec.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_message_codec.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_method_call.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_method_channel.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_method_codec.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_method_response.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_plugin_registrar.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_plugin_registry.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_standard_message_codec.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_standard_method_codec.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_string_codec.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_value.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_view.h,
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/flutter_linux.h, _phony_
[  +10 ms] FAILED: /home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/libflutter_linux_gtk.so
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_basic_message_channel.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_binary_codec.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_binary_messenger.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_dart_project.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_engine.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_json_message_codec.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_json_method_codec.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_message_codec.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_method_call.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_method_channel.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_method_codec.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_method_response.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_plugin_registrar.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_plugin_registry.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_standard_message_codec.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_standard_method_codec.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_string_codec.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_value.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/fl_view.h
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/linux/flutter/ephemeral/flutter_linux/flutter_linux.h flutter/_phony_
[   +2 ms] cd /home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/build/linux/x64/release/flutter &&
/snap/flutter/141/usr/bin/cmake -E env FLUTTER_ROOT=/home/amir/snap/flutter/common/flutter
PROJECT_DIR=/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter DART_OBFUSCATION=false TRACK_WIDGET_CREATION=true
TREE_SHAKE_ICONS=true PACKAGE_CONFIG=/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/.dart_tool/package_config.json
FLUTTER_TARGET=lib/main.dart /home/amir/snap/flutter/common/flutter/packages/flutter_tools/bin/tool_backend.sh linux-x64 Release
[        ] [  +11 ms] Unable to locate an Android SDK.
[        ] [   +8 ms] executing: uname -m
[        ] [   +2 ms] Exit code 0 from: uname -m
[        ] [        ] x86_64
[        ] [  +17 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LegacyCanvasKitRemover' is not required, skipping update.
[        ] [   +1 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] [  +75 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LegacyCanvasKitRemover' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ] [        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ] [        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[        ] [  +34 ms] Initializing file store
[        ] [  +14 ms] Skipping target: gen_localizations
[        ] [   +9 ms] gen_dart_plugin_registrant: Starting due to {InvalidatedReasonKind.inputChanged: The following inputs have updated
contents: /home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/.dart_tool/package_config_subset}
[        ] [   +4 ms] unpack_linux: Starting due to {}
[        ] [  +65 ms] unpack_linux: Complete
[        ] [ +152 ms] Found plugin app_links at /home/amir/.pub-cache/git/app_links-f897e1d81a3d9fcea632588479bc2814d493f7b5/
[        ] [  +22 ms] Found plugin connectivity_plus at /home/amir/.pub-cache/hosted/pub.dev/connectivity_plus-4.0.1/
[        ] [   +4 ms] Found plugin device_info_plus at /home/amir/.pub-cache/hosted/pub.dev/device_info_plus-9.0.1/
[        ] [   +4 ms] Found plugin file_picker at /home/amir/.pub-cache/hosted/pub.dev/file_picker-5.3.1/
[        ] [   +2 ms] Found plugin flutter_plugin_android_lifecycle at
/home/amir/.pub-cache/hosted/pub.dev/flutter_plugin_android_lifecycle-2.0.15/
[        ] [        ] Found plugin fluttertoast at /home/amir/.pub-cache/hosted/pub.dev/fluttertoast-8.2.2/
[        ] [   +3 ms] Found plugin hotkey_manager at /home/amir/.pub-cache/hosted/pub.dev/hotkey_manager-0.1.7/
[        ] [   +7 ms] Found plugin package_info_plus at /home/amir/.pub-cache/hosted/pub.dev/package_info_plus-4.0.1/
[        ] [   +1 ms] Found plugin path_provider at /home/amir/.pub-cache/hosted/pub.dev/path_provider-2.0.15/
[        ] [        ] Found plugin path_provider_android at /home/amir/.pub-cache/hosted/pub.dev/path_provider_android-2.0.27/
[        ] [        ] Found plugin path_provider_foundation at /home/amir/.pub-cache/hosted/pub.dev/path_provider_foundation-2.2.3/
[        ] [        ] Found plugin path_provider_linux at /home/amir/.pub-cache/hosted/pub.dev/path_provider_linux-2.1.11/
[        ] [        ] Found plugin path_provider_windows at /home/amir/.pub-cache/hosted/pub.dev/path_provider_windows-2.1.6/
[        ] [   +4 ms] Found plugin rich_clipboard at /home/amir/.pub-cache/hosted/pub.dev/rich_clipboard-1.0.1/
[   +2 ms] [        ] Found plugin rich_clipboard_android at /home/amir/.pub-cache/hosted/pub.dev/rich_clipboard_android-1.0.1/
[        ] [        ] Found plugin rich_clipboard_ios at /home/amir/.pub-cache/hosted/pub.dev/rich_clipboard_ios-1.0.0/
[        ] [        ] Found plugin rich_clipboard_linux at /home/amir/.pub-cache/hosted/pub.dev/rich_clipboard_linux-1.0.1/
[        ] [        ] Found plugin rich_clipboard_macos at /home/amir/.pub-cache/hosted/pub.dev/rich_clipboard_macos-1.0.1/
[        ] [        ] Found plugin rich_clipboard_web at /home/amir/.pub-cache/hosted/pub.dev/rich_clipboard_web-1.0.2/
[        ] [        ] Found plugin rich_clipboard_windows at /home/amir/.pub-cache/hosted/pub.dev/rich_clipboard_windows-1.0.2/
[        ] [   +2 ms] Found plugin screen_retriever at /home/amir/.pub-cache/hosted/pub.dev/screen_retriever-0.1.6/
[        ] [   +1 ms] Found plugin shared_preferences at /home/amir/.pub-cache/hosted/pub.dev/shared_preferences-2.1.1/
[        ] [        ] Found plugin shared_preferences_android at /home/amir/.pub-cache/hosted/pub.dev/shared_preferences_android-2.1.4/
[        ] [        ] Found plugin shared_preferences_foundation at
/home/amir/.pub-cache/hosted/pub.dev/shared_preferences_foundation-2.2.2/
[        ] [        ] Found plugin shared_preferences_linux at /home/amir/.pub-cache/hosted/pub.dev/shared_preferences_linux-2.2.0/
[        ] [   +2 ms] Found plugin shared_preferences_web at /home/amir/.pub-cache/hosted/pub.dev/shared_preferences_web-2.1.0/
[        ] [        ] Found plugin shared_preferences_windows at /home/amir/.pub-cache/hosted/pub.dev/shared_preferences_windows-2.2.0/
[        ] [   +1 ms] Found plugin sign_in_with_apple at /home/amir/.pub-cache/hosted/pub.dev/sign_in_with_apple-4.3.0/
[        ] [        ] Found plugin sign_in_with_apple_web at /home/amir/.pub-cache/hosted/pub.dev/sign_in_with_apple_web-1.0.1/
[        ] [  +17 ms] Found plugin url_launcher at /home/amir/.pub-cache/hosted/pub.dev/url_launcher-6.1.11/
[        ] [        ] Found plugin url_launcher_android at /home/amir/.pub-cache/hosted/pub.dev/url_launcher_android-6.0.34/
[        ] [        ] Found plugin url_launcher_ios at /home/amir/.pub-cache/hosted/pub.dev/url_launcher_ios-6.1.4/
[        ] [        ] Found plugin url_launcher_linux at /home/amir/.pub-cache/hosted/pub.dev/url_launcher_linux-3.0.5/
[        ] [        ] Found plugin url_launcher_macos at /home/amir/.pub-cache/hosted/pub.dev/url_launcher_macos-3.0.5/
[        ] [   +1 ms] Found plugin url_launcher_web at /home/amir/.pub-cache/hosted/pub.dev/url_launcher_web-2.0.17/
[        ] [        ] Found plugin url_launcher_windows at /home/amir/.pub-cache/hosted/pub.dev/url_launcher_windows-3.0.6/
[        ] [   +5 ms] Found plugin webview_flutter at /home/amir/.pub-cache/hosted/pub.dev/webview_flutter-4.2.1/
[        ] [        ] Found plugin webview_flutter_android at /home/amir/.pub-cache/hosted/pub.dev/webview_flutter_android-3.7.1/
[        ] [        ] Found plugin webview_flutter_wkwebview at /home/amir/.pub-cache/hosted/pub.dev/webview_flutter_wkwebview-3.4.4/
[        ] [   +1 ms] Found plugin window_manager at /home/amir/.pub-cache/hosted/pub.dev/window_manager-0.3.4/
[        ] [   +3 ms] Found plugin appflowy_backend at
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/packages/appflowy_backend/
[        ] [   +1 ms] Found plugin flowy_infra_ui at
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/packages/flowy_infra_ui/
[        ] [        ] Found plugin flowy_infra_ui_web at
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/packages/flowy_infra_ui/flowy_infra_ui_web/
[   +1 ms] [   +4 ms] Found plugin integration_test at /home/amir/snap/flutter/common/flutter/packages/integration_test/
[        ] [  +14 ms] gen_dart_plugin_registrant: Complete
[        ] [        ] kernel_snapshot: Starting due to {}
[   +2 ms] [   +8 ms] /home/amir/snap/flutter/common/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev
/home/amir/snap/flutter/common/flutter/bin/cache/dart-sdk/bin/snapshots/frontend_server.dart.snapshot --sdk-root
/home/amir/snap/flutter/common/flutter/bin/cache/artifacts/engine/common/flutter_patched_sdk_product/ --target=flutter
--no-print-incremental-dependencies -Ddart.vm.profile=false -Ddart.vm.product=true --aot --tfa --target-os linux --packages
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/.dart_tool/package_config.json --output-dill
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/.dart_tool/flutter_build/8a507e605ba3f73fec5569230ed0d6be/app.dill
--depfile
/home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/.dart_tool/flutter_build/8a507e605ba3f73fec5569230ed0d6be/kernel_snapsh
ot.d --source
file:///home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/.dart_tool/flutter_build/dart_plugin_registrant.dart --source
package:flutter/src/dart_plugin_registrant.dart
-Dflutter.dart_plugin_registrant=file:///home/amir/projects/personal/code/AppFlowy/frontend/appflowy_flutter/.dart_tool/flutter_build/dart_p
lugin_registrant.dart --verbosity=error package:appflowy/main.dart
[   +1 ms] [+11018 ms] packages/flowy_infra_ui/lib/widget/dialog/styled_dialogs.dart:74:27: Error: The argument type 'BorderRadius?' can't
be assigned to the parameter type 'BorderRadiusGeometry' because 'BorderRadius?' is nullable and 'BorderRadiusGeometry' isn't.
[   +1 ms] [   +1 ms]  - 'BorderRadius' is from 'package:flutter/src/painting/border_radius.dart'
('../../../../../../snap/flutter/common/flutter/packages/flutter/lib/src/painting/border_radius.dart').
[        ] [        ]  - 'BorderRadiusGeometry' is from 'package:flutter/src/painting/border_radius.dart'
('../../../../../../snap/flutter/common/flutter/packages/flutter/lib/src/painting/border_radius.dart').
[        ] [        ]             borderRadius: borderRadius,
[        ] [        ]                           ^
[        ] [+9369 ms] Persisting file store
[        ] [   +1 ms] Done persisting file store
[        ] [   +1 ms] Target kernel_snapshot failed: Exception
[        ]            #0      KernelSnapshot.build (package:flutter_tools/src/build_system/targets/common.dart:256:7)
[        ]            <asynchronous suspension>
[        ]            #1      _BuildInstance._invokeInternal (package:flutter_tools/src/build_system/build_system.dart:853:9)
[        ]            <asynchronous suspension>
[        ]            #2      Future.wait.<anonymous closure> (dart:async/future.dart:525:21)
[        ]            <asynchronous suspension>
[        ]            #3      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:791:32)
[        ]            <asynchronous suspension>
[        ]            #4      Future.wait.<anonymous closure> (dart:async/future.dart:525:21)
[        ]            <asynchronous suspension>
[        ]            #5      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:791:32)
[        ]            <asynchronous suspension>
[        ]            #6      Future.wait.<anonymous closure> (dart:async/future.dart:525:21)
[        ]            <asynchronous suspension>
[        ]            #7      _BuildInstance.invokeTarget (package:flutter_tools/src/build_system/build_system.dart:791:32)
[   +1 ms]            <asynchronous suspension>
[        ]            #8      FlutterBuildSystem.build (package:flutter_tools/src/build_system/build_system.dart:620:16)
[        ]            <asynchronous suspension>
[        ]            #9      AssembleCommand.runCommand (package:flutter_tools/src/commands/assemble.dart:315:32)
[        ]            <asynchronous suspension>
[        ]            #10     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1297:27)
[        ]            <asynchronous suspension>
[        ]            #11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]            <asynchronous suspension>
[        ]            #12     CommandRunner.runCommand (package:args/command_runner.dart:212:13)
[        ]            <asynchronous suspension>
[        ]            #13     FlutterCommandRunner.runCommand.<anonymous closure>
(package:flutter_tools/src/runner/flutter_command_runner.dart:339:9)
[        ]            <asynchronous suspension>
[        ]            #14     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]            <asynchronous suspension>
[        ]            #15     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:285:5)
[        ]            <asynchronous suspension>
[        ]            #16     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:115:9)
[        ]            <asynchronous suspension>
[        ]            #17     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]            <asynchronous suspension>
[        ]            #18     main (package:flutter_tools/executable.dart:90:3)
[        ]            <asynchronous suspension>
[        ] [   +6 ms] "flutter assemble" took 20,896ms.
[        ] [   +2 ms] 
[        ]            #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
[        ]            #1      AssembleCommand.runCommand (package:flutter_tools/src/commands/assemble.dart:332:7)
[        ]            <asynchronous suspension>
[        ]            #2      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1297:27)
[        ]            <asynchronous suspension>
[        ]            #3      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]            <asynchronous suspension>
[        ]            #4      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
[        ]            <asynchronous suspension>
[        ]            #5      FlutterCommandRunner.runCommand.<anonymous closure>
(package:flutter_tools/src/runner/flutter_command_runner.dart:339:9)
[        ]            <asynchronous suspension>
[        ]            #6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]            <asynchronous suspension>
[        ]            #7      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:285:5)
[        ]            <asynchronous suspension>
[        ]            #8      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:115:9)
[        ]            <asynchronous suspension>
[        ]            #9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
[        ]            <asynchronous suspension>
[        ]            #10     main (package:flutter_tools/executable.dart:90:3)
[        ]            <asynchronous suspension>
[        ] [ +256 ms] ensureAnalyticsSent: 251ms
[        ] [        ] Running 0 shutdown hooks
[        ] [        ] Shutdown hooks complete
[        ] [        ] exiting with code 1
[        ] ninja: build stopped: subcommand failed.
[  +11 ms] Building Linux application... (completed in 22.2s)
[        ] "flutter linux" took 22,825ms.
[   +5 ms] Build process failed
[        ] 
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
           #1      _runBuild (package:flutter_tools/src/linux/build_linux.dart:181:5)
           <asynchronous suspension>
           #2      buildLinux (package:flutter_tools/src/linux/build_linux.dart:75:5)
           <asynchronous suspension>
           #3      BuildLinuxCommand.runCommand (package:flutter_tools/src/commands/build_linux.dart:86:5)
           <asynchronous suspension>
           #4      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1297:27)
           <asynchronous suspension>
           #5      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #6      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
           <asynchronous suspension>
           #7      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:339:9)
           <asynchronous suspension>
           #8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #9      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:285:5)
           <asynchronous suspension>
           #10     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:115:9)
           <asynchronous suspension>
           #11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #12     main (package:flutter_tools/executable.dart:90:3)
           <asynchronous suspension>
           
           
[ +253 ms] ensureAnalyticsSent: 250ms
[        ] Running 0 shutdown hooks
[        ] Shutdown hooks complete
[        ] exiting with code 1
[cargo-make][1] ERROR - Error while executing command, exit code: 1
[cargo-make][1] WARN - Build Failed.
[cargo-make] INFO - Running Task: legacy-migration
[cargo-make] INFO - Running Task: restore-crate-type
[cargo-make] ERROR - Task error detected, exit code: 1
[cargo-make] WARN - Build Failed.`

--- a/frontend/appflowy_flutter/lib/startup/tasks/app_widget.dart
+++ b/frontend/appflowy_flutter/lib/startup/tasks/app_widget.dart
@@ -56,6 +56,7 @@ class InitAppWidgetTask extends LaunchTask {
           Locale('tr', 'TR'),
           Locale('zh', 'CN'),
           Locale('zh', 'TW'),
+          Locale('fa'),
         ],
         path: 'assets/translations',
         fallbackLocale: const Locale('en'),

--- a/frontend/appflowy_flutter/packages/flowy_infra/lib/language.dart
+++ b/frontend/appflowy_flutter/packages/flowy_infra/lib/language.dart
@@ -55,6 +55,8 @@ String languageFromLocale(Locale locale) {
       return "Svenska";
     case "tr":
       return "Türkçe";
+    case "fa":
+      return "فارسی";
 
     // If not found then the language code will be displayed
     default:

--- a/frontend/resources/translations/fa.json
+++ b/frontend/resources/translations/fa.json
@@ -1,0 +1,678 @@
+{
+  "appName": "AppFlowy",
+  "defaultUsername": "من",
+  "welcomeText": "به @:appName خوش آمدید",
+  "githubStarText": "به گیت‌هاب ما ستاره دهید",
+  "subscribeNewsletterText": "اشتراک در خبرنامه",
+  "letsGoButtonText": "شروع کنید",
+  "title": "عنوان",
+  "youCanAlso": "همچنین می‌توانید",
+  "and": "و",
+  "blockActions": {
+    "addBelowTooltip": "برای افزودن در زیر کلیک کنید",
+    "addAboveCmd": "Alt+click",
+    "addAboveMacCmd": "Option+click",
+    "addAboveTooltip": "برای افزودن در بالا",
+    "dragTooltip": "برای حرکت بکشید",
+    "openMenuTooltip": "برای باز کردن منو کلیک کنید"
+  },
+  "signUp": {
+    "buttonText": "ثبت نام",
+    "title": "ثبت نام در @:appName",
+    "getStartedText": "شروع کنید",
+    "emptyPasswordError": "رمز عبور نمی تواند خالی باشد",
+    "repeatPasswordEmptyError": "تکرار رمز عبور نمی‌تواند خالی باشد",
+    "unmatchedPasswordError": "تکرار رمز عبور مشابه رمز عبور نیست",
+    "alreadyHaveAnAccount": "از قبل حساب دارید؟",
+    "emailHint": "ایمیل",
+    "passwordHint": "رمز عبور",
+    "repeatPasswordHint": "تکرار رمز عبور",
+    "signUpWith": "ثبت نام با:"
+  },
+  "signIn": {
+    "loginTitle": "ورود به @:appName",
+    "loginButtonText": "ورود",
+    "loginAsGuestButtonText": "شروع کنید",
+    "continueAnonymousUser": "ادامه دادن به صورت کاربر مهمان",
+    "buttonText": "ورود",
+    "forgotPassword": "رمز عبور را فراموش کرده اید؟",
+    "emailHint": "ایمیل",
+    "passwordHint": "رمز عبور",
+    "dontHaveAnAccount": "آیا حساب کاربری ندارید؟",
+    "repeatPasswordEmptyError": "تکرار رمز عبور نمی‌تواند خالی باشد",
+    "unmatchedPasswordError": "تکرار رمز عبور مشابه رمز عبور نیست",
+    "signInWith": "ثبت نام با:"
+  },
+  "workspace": {
+    "create": "ایجاد فضای کار",
+    "hint": "فضای کار",
+    "notFoundError": "فضای کاری پیدا نشد"
+  },
+  "shareAction": {
+    "buttonText": "اشتراک گذاری",
+    "workInProgress": "به زودی",
+    "markdown": "Markdown",
+    "copyLink": "کپی کردن لینک"
+  },
+  "moreAction": {
+    "small": "کوچک",
+    "medium": "متوسط",
+    "large": "بزرگ",
+    "fontSize": "اندازه قلم",
+    "import": "اضافه کردن",
+    "moreOptions": "گزینه های بیشتر"
+  },
+  "importPanel": {
+    "textAndMarkdown": "Text & Markdown",
+    "documentFromV010": "سند از نسخه 0.1.0",
+    "databaseFromV010": "پایگاه داده از نسخه 0.1.0",
+    "csv": "CSV",
+    "database": "پایگاه داده"
+  },
+  "disclosureAction": {
+    "rename": "تغییر نام",
+    "delete": "حذف",
+    "duplicate": "تکرار کردن",
+    "unfavorite": "حذف از موارد دلخواه",
+    "favorite": "افزودن به موارد دلخواه",
+    "openNewTab": "باز کردن در یک برگه جدید",
+    "moveTo": "انتقال به",
+    "addToFavorites": "افزودن به موارد دلخواه",
+    "copyLink": "کپی کردن لینک"
+  },
+  "blankPageTitle": "صفحه خالی",
+  "newPageText": "صفحه جدید",
+  "trash": {
+    "text": "سطل زباله",
+    "restoreAll": "بازیابی همه",
+    "deleteAll": "حذف همه",
+    "pageHeader": {
+      "fileName": "نام فایل",
+      "lastModified": "آخرین بازنگری",
+      "created": "ایجاد شده"
+    },
+    "confirmDeleteAll": {
+      "title": "آیا می‌خواهید که همه صفحه‌ها را در سطل زباله حذف کنید؟",
+      "caption": "این عمل قابل بازگشت نیست."
+    },
+    "confirmRestoreAll": {
+      "title": "آیا می‌خواهید که همه صفحه‌ها را در سطل زباله بازیابی کنید؟",
+      "caption": "این عمل قابل بازگشت نیست."
+    }
+  },
+  "deletePagePrompt": {
+    "text": "این صفحه در سطل زباله است",
+    "restore": "بازیابی صفحه",
+    "deletePermanent": "حذف دائمی"
+  },
+  "dialogCreatePageNameHint": "نام صفحه",
+  "questionBubble": {
+    "shortcuts": "میانبرها",
+    "whatsNew": "تازه‌ترین‌ها",
+    "help": "پشتیبانی و مستندات",
+    "markdown": "Markdown",
+    "debug": {
+      "name": "اطلاعات اشکال‌زدایی",
+      "success": "طلاعات اشکال زدایی در کلیپ بورد کپی شد!",
+      "fail": "نمی توان اطلاعات اشکال زدایی را در کلیپ بورد کپی کرد"
+    },
+    "feedback": "بازخورد"
+  },
+  "menuAppHeader": {
+    "moreButtonToolTip": "حذف، تغییر نام، و موارد دیگر...",
+    "addPageTooltip": "یک صفحه در داخل اضافه کنید",
+    "defaultNewPageName": "بدون عنوان",
+    "renameDialog": "تغییر نام"
+  },
+  "toolbar": {
+    "undo": "Undo",
+    "redo": "Redo",
+    "bold": "پررنگ",
+    "italic": "ایتالیک",
+    "underline": "با خط در زیر",
+    "strike": "با خط در وسط",
+    "numList": "فهرست شماره‌گذاری شده",
+    "bulletList": "فهرست موردی",
+    "checkList": "چک‌لیست",
+    "inlineCode": "کد درونی",
+    "quote": "Quote Block",
+    "header": "سربرگ",
+    "highlight": "برجسته کردن",
+    "color": "رنگ",
+    "addLink": "افزودن لینک",
+    "link": "لینک"
+  },
+  "tooltip": {
+    "lightMode": "تغییر به مود روشن",
+    "darkMode": "تغییر به مود تاریک",
+    "openAsPage": "باز کردن به عنوان صفحه",
+    "addNewRow": "اضافه کردن سطر جدید",
+    "openMenu": "برای باز کردن منو کلیک کنید",
+    "dragRow": "برای مرتب کردن مجدد ردیف فشار طولانی دهید",
+    "viewDataBase": "مشاهده پایگاه داده",
+    "referencePage": "این {name} ارجاع داده شده است",
+    "addBlockBelow": "یک بلوک در زیر اضافه کنید"
+  },
+  "sideBar": {
+    "closeSidebar": "بستن نوار کناری",
+    "openSidebar": "باز کردن نوار کناری",
+    "personal": "شخصی",
+    "favorites": "مورد علاقه",
+    "clickToHidePersonal": "برای پنهان کردن قسمت شخصی کلیک کنید",
+    "clickToHideFavorites": "برای پنهان کردن بخش دلخواه کلیک کنید",
+    "addAPage": "افزودن یک صفحه"
+  },
+  "notifications": {
+    "export": {
+      "markdown": "متن به یادداشت تبدیل شود",
+      "path": "Documents/flowy"
+    }
+  },
+  "contactsPage": {
+    "title": "مخاطبین",
+    "whatsHappening": "این هفته چه اتفاقی می‌افتد؟",
+    "addContact": "افزودن مخاطب",
+    "editContact": "ویرایش مخاطب"
+  },
+  "button": {
+    "OK": "باشه",
+    "Done": "انجام شد",
+    "Cancel": "لغو",
+    "signIn": "ورود",
+    "signOut": "خروج",
+    "complete": "کامل شد",
+    "save": "ذخیره‌سازی",
+    "generate": "تولید‌کردن",
+    "esc": "ESC",
+    "keep": "نگه داشتن",
+    "tryAgain": "دوباره تلاش کنید",
+    "discard": "در نظر نگرفتن",
+    "replace": "جایگزین کردن",
+    "insertBelow": "جاگذاری در پایین",
+    "upload": "بارگذاری",
+    "edit": "ویرایش",
+    "delete": "حذف کردن",
+    "duplicate": "تکرار کردن",
+    "done": "انجام شد",
+    "putback": "بازگشت"
+  },
+  "label": {
+    "welcome": "خوش آمدید!",
+    "firstName": "نام",
+    "middleName": "نام میانی",
+    "lastName": "نام خانوادگی",
+    "stepX": "Step {X}"
+  },
+  "oAuth": {
+    "err": {
+      "failedTitle": "امکان اتصال به حساب شما وجود ندارد.",
+      "failedMsg": "لطفا مطمئن شوید که فرآیند ورود را در مرورگر خود تکمیل کرده اید."
+    },
+    "google": {
+      "title": "ورود با اکانت گوگل",
+      "instruction1": "برای دسترسی به مخاطبان خود در گوگل، می‌بایست به به این برنامه از طریق مرورگر خود دسترسی دهید.",
+      "instruction2": "این کد را با کلیک کردن روی آیکون یا انتخاب متن در کلیپ بورد خود کپی کنید:",
+      "instruction3": "به لینک زیر در مرورگر وب خود بروید و کد بالا را وارد کنید:",
+      "instruction4": "پس از تکمیل ثبت نام، دکمه زیر را فشار دهید:"
+    }
+  },
+  "settings": {
+    "title": "تنظیمات",
+    "menu": {
+      "appearance": "ظاهر برنامه",
+      "language": "زبان‌ها",
+      "user": "کاربر",
+      "files": "فایل‌ها",
+      "open": "باز کردن تنظیمات",
+      "logout": "خروج",
+      "logoutPrompt": "آیا مطمئن هستید که می‌خواهید خارج شوید؟",
+      "syncSetting": "تنظیمات همگام‌سازی",
+      "enableSync": "فعال کردن همگام‌سازی",
+      "historicalUserList": "سابقه ورود کاربر",
+      "historicalUserListTooltip": "این لیست اکانت‌های ناشناس شما را نمایش می‌دهد. می‌توانید روی یک حساب برای مشاهده جزییات آن کلیک کنید. حساب‌های ناشناس با کلیک کردن روی دکمه شروع‌کنید ایجاد می‌شوند",
+      "openHistoricalUser": "برای باز کردن حساب ناشناس کلیک کنید"
+    },
+    "appearance": {
+      "resetSetting": "تنظیم کردن از اول",
+      "fontFamily": {
+        "label": "خانواده فونت",
+        "search": "جستجو"
+      },
+      "themeMode": {
+        "label": "حالت تم",
+        "light": "حالت روشن",
+        "dark": "حالت تاریک",
+        "system": "اعمال حالت"
+      },
+      "themeUpload": {
+        "button": "بارگذاری",
+        "description": "تم قالب AppFlowy خود را با استفاده از دکمه زیر آپلود کنید.",
+        "failure": "تم قالب آپلود شده نامعتبر است.",
+        "loading": "لطفاً منتظر بمانید تا تم قالب شما را اعتبارسنجی و آپلود کنیم...",
+        "uploadSuccess": "تم قالب شما با موفقیت آپلود شد",
+        "deletionFailure": "تم حذف نشد. سعی کنید آن را به صورت دستی حذف کنید.",
+        "filePickerDialogTitle": "یک فایل .flowy_plugin را انتخاب کنید",
+        "urlUploadFailure": "نشانی اینترنتی باز نشد: {}"
+      },
+      "theme": "تم قالب",
+      "builtInsLabel": "قالب‌های پیش‌ساخته",
+      "pluginsLabel": "پلاگین‌ها"
+    },
+    "files": {
+      "copy": "کپی",
+      "defaultLocation": "خواندن فایل‌ها و مکان ذخیره داده‌ها",
+      "exportData": "از داده‌های خود خروجی بگیرید",
+      "doubleTapToCopy": "برای کپی کردن دوبار کلیک کنید",
+      "restoreLocation": "بازیابی به مسیر پیش فرض AppFlowy",
+      "customizeLocation": "پوشه دیگری باز کنید",
+      "restartApp": "لطفاً برنامه را مجدداً راه اندازی کنید تا تغییرات اعمال شوند.",
+      "exportDatabase": "از پایگاه داده‌ها خروجی بگیرید",
+      "selectFiles": "فایل‌هایی را که می‌خواهید از آنها خروجی بگیرید انتخاب کنید",
+      "selectAll": "انتخاب همه",
+      "deselectAll": "لغو انتخاب همه",
+      "createNewFolder": "ایجاد یک پوشه جدید",
+      "createNewFolderDesc": "کجا می خواهید داده‌های خود را ذخیره کنید",
+      "defineWhereYourDataIsStored": "محل ذخیره داده های خود را مشخص کنید",
+      "open": "باز کردن",
+      "openFolder": "باز کردن یک پوشه موجود",
+      "openFolderDesc": "خواندن و نوشتن آن در یک پوشه AppFlowy موجود",
+      "folderHintText": "نام پوشه",
+      "location": "ایجاد یک پوشه جدید",
+      "locationDesc": "یک نام برای پوشه داده AppFlowy خود انتخاب کنید",
+      "browser": "مرورگر",
+      "create": "ایجاد کردن",
+      "set": "تنظیم کردن",
+      "folderPath": "مسیری برای ذخیره پوشه",
+      "locationCannotBeEmpty": "مسیر نمی‌تواند خالی باشد",
+      "pathCopiedSnackbar": "مسیر ذخیره‌سازی فایل در کلیپ‌بورد کپی شد!",
+      "changeLocationTooltips": "تغییر دایرکتوری داده",
+      "change": "تغییر",
+      "openLocationTooltips": "باز کردن یک فهرست پوشه دیگر",
+      "openCurrentDataFolder": "باز کردن فهرست پوشه فعلی",
+      "recoverLocationTooltips": "بازنشانی به فهرست داده های پیش فرض AppFlowy",
+      "exportFileSuccess": "خروجی گرفتن از فایل با موفقیت انجام شد.",
+      "exportFileFail": "خروجی گرفتن از فایل انجام نشد!",
+      "export": "خروجی گرفتن"
+    },
+    "user": {
+      "name": "نام",
+      "icon": "آیکون",
+      "selectAnIcon": "انتخاب یک آیکون",
+      "pleaseInputYourOpenAIKey": "لطفا کلید OpenAI خود را وارد کنید",
+      "clickToLogout": "برای خروج از کاربر فعلی کلیک کنید"
+    },
+    "shortcuts": {
+      "shortcutsLabel": "میانبرها",
+      "command": "دستور",
+      "keyBinding": "میانبرهای کیبورد",
+      "addNewCommand": "اضافه کردن فرمان جدید",
+      "updateShortcutStep": "کلید ترکیبی دلخواه را فشار دهید و ENTER را فشار دهید",
+      "shortcutIsAlreadyUsed": "این میانبر قبلاً برای: {conflict} استفاده شده است",
+      "resetToDefault": "بازگشت به میانبرهای پیش‌فرض",
+      "couldNotLoadErrorMsg": "میانبرها بارگذاری نشد، دوباره امتحان کنید",
+      "couldNotSaveErrorMsg": "میانبرها ذخیره نشد، دوباره امتحان کنید"
+    }
+  },
+  "grid": {
+    "deleteView": "آیا مطمئن هستید که می خواهید این نما را حذف کنید؟",
+    "createView": "جدید",
+    "settings": {
+      "filter": "فیلتر",
+      "sort": "مرتب کردن",
+      "sortBy": "مرتب سازی بر اساس",
+      "properties": "ویژگی‌ها",
+      "reorderPropertiesTooltip": "برای مرتب کردن مجدد بکشید",
+      "group": "گروه",
+      "addFilter": "افزودن فیلتر",
+      "deleteFilter": "حذف فیلتر",
+      "filterBy": "فیلتر بر اساس...",
+      "typeAValue": "یک مقدار را تایپ کنید...",
+      "layout": "طرح‌بندی",
+      "databaseLayout": "طرح‌بندی"
+    },
+    "textFilter": {
+      "contains": "شامل",
+      "doesNotContain": "شامل نمی‌شود",
+      "endsWith": "پایان با",
+      "startWith": "شروع با",
+      "is": "هست",
+      "isNot": "نیست",
+      "isEmpty": "خالی است",
+      "isNotEmpty": "خالی نیست",
+      "choicechipPrefix": {
+        "isNot": "مخالف",
+        "startWith": "شروع با",
+        "endWith": "پایان با",
+        "isEmpty": "خالی است",
+        "isNotEmpty": "خالی نیست"
+      }
+    },
+    "checkboxFilter": {
+      "isChecked": "بررسی شده",
+      "isUnchecked": "بررسی نشده",
+      "choicechipPrefix": {
+        "is": "است"
+      }
+    },
+    "checklistFilter": {
+      "isComplete": "کامل است",
+      "isIncomplted": "کامل نیست"
+    },
+    "singleSelectOptionFilter": {
+      "is": "است",
+      "isNot": "نیست",
+      "isEmpty": "خالی است",
+      "isNotEmpty": "خالی نیست"
+    },
+    "multiSelectOptionFilter": {
+      "contains": "شامل",
+      "doesNotContain": "شامل نیست",
+      "isEmpty": "خالی است",
+      "isNotEmpty": "خالی نیست"
+    },
+    "field": {
+      "hide": "پنهان کردن",
+      "insertLeft": "درج در چپ",
+      "insertRight": "درج در راست",
+      "duplicate": "تکرار کردن",
+      "delete": "حذف کردن",
+      "textFieldName": "متن",
+      "checkboxFieldName": "موارد انتخابی",
+      "dateFieldName": "تاریخ",
+      "updatedAtFieldName": "آخرین زمان بازنگری",
+      "createdAtFieldName": "زمان ایجاد",
+      "numberFieldName": "شماره‌ها",
+      "singleSelectFieldName": "انتخاب",
+      "multiSelectFieldName": "چند‌انتخابی",
+      "urlFieldName": "نشانی اینترنتی",
+      "checklistFieldName": "چک لیست",
+      "numberFormat": "قالب شماره",
+      "dateFormat": "قالب تاریخ",
+      "includeTime": "شامل کردن زمان",
+      "dateFormatFriendly": "Month Day, Year",
+      "dateFormatISO": "Year-Month-Day",
+      "dateFormatLocal": "Month/Day/Year",
+      "dateFormatUS": "Year/Month/Day",
+      "dateFormatDayMonthYear": "Day/Month/Year",
+      "timeFormat": "قالب زمان",
+      "invalidTimeFormat": "قالب نامعتبر",
+      "timeFormatTwelveHour": "دوازده ساعته",
+      "timeFormatTwentyFourHour": "بیست‌و‌چهار ساعته",
+      "clearDate": "پاک کردن",
+      "addSelectOption": "افزودن یک گزینه",
+      "optionTitle": "گزینه‌ها",
+      "addOption": "افزودن گزینه",
+      "editProperty": "ویرایش ویژگی",
+      "newProperty": "ویژگی جدید",
+      "deleteFieldPromptMessage": "آیا مطمئن هستید؟ این ویژگی حذف خواهد شد",
+      "newColumn": "ستون جدید"
+    },
+    "sort": {
+      "ascending": "صعودی",
+      "descending": "نزولی",
+      "deleteAllSorts": "حذف همه مرتب‌سازی‌ها",
+      "addSort": "اضافه کردن مرتب‌سازی"
+    },
+    "row": {
+      "duplicate": "تکرار کردن",
+      "delete": "حذف کردن",
+      "textPlaceholder": "خالی",
+      "copyProperty": "ویژگی در کلیپ‌بورد کپی شد.",
+      "count": "شمارش",
+      "newRow": "سطر جدید",
+      "action": "اعمال"
+    },
+    "selectOption": {
+      "create": "ایجاد",
+      "purpleColor": "بنفش",
+      "pinkColor": "صورتی",
+      "lightPinkColor": "صورتی روشن",
+      "orangeColor": "نارنجی",
+      "yellowColor": "زرد",
+      "limeColor": "لیمویی",
+      "greenColor": "سبز",
+      "aquaColor": "آکوا",
+      "blueColor": "آبی",
+      "deleteTag": "حذف برچسب",
+      "colorPanelTitle": "رنگ‌ها",
+      "panelTitle": "یک گزینه انتخاب یا ایجاد کنید.",
+      "searchOption": "جستجوی یک گزینه"
+    },
+    "checklist": {
+      "panelTitle": "یک مورد اضافه کنید"
+    },
+    "menuName": "شبکه‌ای",
+    "referencedGridPrefix": "نمایش"
+  },
+  "document": {
+    "menuName": "سند",
+    "date": {
+      "timeHintTextInTwelveHour": "01:00 PM",
+      "timeHintTextInTwentyFourHour": "13:00"
+    },
+    "slashMenu": {
+      "board": {
+        "selectABoardToLinkTo": "یک بورد برای لینک کردن انتخاب کنید.",
+        "createANewBoard": "ایجاد یک بورد جدید"
+      },
+      "grid": {
+        "selectAGridToLinkTo": "یک شبکه‌ نمایش برای لینک کردن انتخاب کنید.",
+        "createANewGrid": "ایجاد یک شبکه نمایش جدید"
+      },
+      "calendar": {
+        "selectACalendarToLinkTo": "یک تقویم برای لینک کردن انتخاب کنید.",
+        "createANewCalendar": "ایجاد یک تقویم جدید"
+      }
+    },
+    "selectionMenu": {
+      "outline": "طرح کلی"
+    },
+    "plugins": {
+      "referencedBoard": "بورد مرجع",
+      "referencedGrid": "شبکه‌نمایش مرجع",
+      "referencedCalendar": "تقویم مرجع",
+      "autoGeneratorMenuItemName": "OpenAI نویسنده",
+      "autoGeneratorTitleName": "از هوش مصنوعی بخواهید هر چیزی بنویسد...",
+      "autoGeneratorLearnMore": "بیشتر بدانید",
+      "autoGeneratorGenerate": "بنویس",
+      "autoGeneratorHintText": "از OpenAI بپرسید ...",
+      "autoGeneratorCantGetOpenAIKey": "کلید OpenAI را نمی توان دریافت کرد",
+      "autoGeneratorRewrite": "بازنویس",
+      "smartEdit": "دستیاران هوشمند",
+      "openAI": "OpenAI",
+      "smartEditFixSpelling": "اصلاح نگارش",
+      "warning": "⚠️ پاسخ‌های هوش مصنوعی می‌توانند نادرست یا گمراه‌کننده باشند",
+      "smartEditSummarize": "خلاصه‌نویسی",
+      "smartEditImproveWriting": "بهبود نگارش",
+      "smartEditMakeLonger": "به نوشته اضافه کن",
+      "smartEditCouldNotFetchResult": "نتیجه‌ای از OpenAI گرفته نشد",
+      "smartEditCouldNotFetchKey": "کلید OpenAI واکشی نشد",
+      "smartEditDisabled": "به OpenAI در تنظیمات وصل شوید",
+      "discardResponse": "آیا می خواهید پاسخ های هوش مصنوعی را حذف کنید؟",
+      "createInlineMathEquation": "ایجاد معادله",
+      "toggleList": "Toggle لیست",
+      "cover": {
+        "changeCover": "تغییر جلد",
+        "colors": "رنگ‌ها",
+        "images": "تصویر‌ها",
+        "clearAll": "پاک کردن همه",
+        "abstract": "چکیده",
+        "addCover": "افزودن جلد",
+        "addLocalImage": "افزودن تصویر",
+        "invalidImageUrl": "مسیر تصویر نامعتبر است",
+        "failedToAddImageToGallery": "افزودن تصویر به گالری انجام نشد",
+        "enterImageUrl": "مسیر تصویر را وارد کنید",
+        "add": "افزودن",
+        "back": "بازگشت",
+        "saveToGallery": "ذخیره در گالری",
+        "removeIcon": "حذف Icon",
+        "pasteImageUrl": "وارد کردن مسیر تصویر",
+        "or": "یا",
+        "pickFromFiles": "انتخاب از فایل‌ها",
+        "couldNotFetchImage": "تصویر واکشی نشد",
+        "imageSavingFailed": "ذخیره تصویر انجام نشد",
+        "addIcon": "افزودن آیکون",
+        "coverRemoveAlert": "پس از حذف از روی جلد حذف خواهد شد.",
+        "alertDialogConfirmation": "آیا مطمئن هستید که می‌خواهید ادامه دهید؟"
+      },
+      "mathEquation": {
+        "addMathEquation": "اضافه کردن معادله ریاضی",
+        "editMathEquation": "ویرایش کردن معادله ریاضی"
+      },
+      "optionAction": {
+        "click": "کلیک کنید",
+        "toOpenMenu": " برای باز کردن منو",
+        "delete": "حذف کردن",
+        "duplicate": "تکرار کردن",
+        "turnInto": "تبدیل به",
+        "moveUp": "بالا بردن",
+        "moveDown": "پایین آوردن",
+        "color": "رنگ",
+        "align": "هم‌تراز کردن",
+        "left": "چپ",
+        "center": "وسط",
+        "right": "راست",
+        "defaultColor": "پیش فرض"
+      },
+      "image": {
+        "copiedToPasteBoard": "لینک تصویر در کلیپ‌بورد کپی شده است"
+      },
+      "outline": {
+        "addHeadingToCreateOutline": "برای ایجاد فهرست مطالب سر‌فصل‌ها را وارد کنید"
+      }
+    },
+    "textBlock": {
+      "placeholder": "برای دستورات '/' را تایپ کنید"
+    },
+    "title": {
+      "placeholder": "بدون عنوان"
+    },
+    "imageBlock": {
+      "placeholder": "برای افزودن تصویر کلیک کنید",
+      "upload": {
+        "label": "بارگذاری",
+        "placeholder": "برای بارگذاری تصویر کلیک کنید"
+      },
+      "url": {
+        "label": "لینک تصویر",
+        "placeholder": "لینک تصویر را وارد کنید"
+      },
+      "support": "محدودیت اندازه تصویر 5 مگابایت است. فرمت‌های پشتیبانی شده: JPEG، PNG، GIF، SVG",
+      "error": {
+        "invalidImage": "تصویر نامعتبر",
+        "invalidImageSize": "اندازه تصویر باید کمتر از 5 مگابایت باشد",
+        "invalidImageFormat": "فرمت تصویر پشتیبانی نمی‌شود. فرمت‌های پشتیبانی شده: JPEG، PNG، GIF، SVG",
+        "invalidImageUrl": "مسیر تصویر نامعتبر است"
+      }
+    },
+    "codeBlock": {
+      "language": {
+        "label": "زبان",
+        "placeholder": "انتخاب زبان"
+      }
+    },
+    "inlineLink": {
+      "placeholder": "پیست کنید یا مسیر را تایپ کنید",
+      "openInNewTab": "باز کردن در برگه جدید",
+      "copyLink": "کپی لینک",
+      "removeLink": "حذف لینک",
+      "url": {
+        "label": "لینک",
+        "placeholder": "لینک را وارد کنید"
+      },
+      "title": {
+        "label": "عنوان لینک",
+        "placeholder": "عنوان لینک را وارد کنید"
+      }
+    },
+    "mention": {
+      "placeholder": "یک شخص یا یک صفحه یا تاریخ را ذکر کنید...",
+      "page": {
+        "label": "لینک به صفحه",
+        "tooltip": "برای باز کردن صفحه کلیک کنید"
+      }
+    }
+  },
+  "board": {
+    "column": {
+      "create_new_card": "ایجاد"
+    },
+    "menuName": "بورد",
+    "referencedBoardPrefix": "نمای"
+  },
+  "calendar": {
+    "menuName": "تقویم",
+    "defaultNewCalendarTitle": "بدون عنوان",
+    "navigation": {
+      "today": "امروز",
+      "jumpToday": "برو به امروز",
+      "previousMonth": "ماه قبل",
+      "nextMonth": "ماه بعد"
+    },
+    "settings": {
+      "showWeekNumbers": "نمایش اعداد هفته",
+      "showWeekends": "نمایش تعطیلات آخر هفته",
+      "firstDayOfWeek": "شروع هفته در",
+      "layoutDateField": "طرح‌بندی تقویم با",
+      "noDateTitle": "بدون تاریخ",
+      "noDateHint": "رویدادهای برنامه‌ریزی نشده در اینجا نشان داده می‌شوند",
+      "clickToAdd": "برای افزودن به تقویم کلیک کنید",
+      "name": "طرح‌بندی تقویم"
+    },
+    "referencedCalendarPrefix": "نمای"
+  },
+  "errorDialog": {
+    "title": "خطای AppFlowy",
+    "howToFixFallback": "بابت مشکل پیش آمده متأسفیم! مشکل و شرح آن را در صفحه GitHub ما ارسال کنید.",
+    "github": "مشاهده در GitHub"
+  },
+  "search": {
+    "label": "جستجو",
+    "placeholder": {
+      "actions": "جستجوی اعمال..."
+    }
+  },
+  "message": {
+    "copy": {
+      "success": "کپی شد!",
+      "fail": "نمی‌توان کپی کرد"
+    }
+  },
+  "unSupportBlock": "نسخه فعلی از این بلوک پشتیبانی نمی‌کند.",
+  "views": {
+    "deleteContentTitle": "آیا مطمئن هستید که می‌خواهید {pageType} را حذف کنید؟",
+    "deleteContentCaption": "اگر این {pageType} را حذف کنید، می‌توانید آن را از سطل زباله بازیابی کنید."
+  },
+  "colors": {
+    "custom": "سفارشی",
+    "default": "پیش‌فرض",
+    "red": "قرمز",
+    "orange": "نارنجی",
+    "yellow": "زرد",
+    "green": "سبز",
+    "blue": "آبی",
+    "purple": "بنفش",
+    "pink": "صورتی",
+    "brown": "قهوه‌ای",
+    "gray": "خاکستری"
+  },
+  "emoji": {
+    "filter": "فیلتر",
+    "random": "تصادفی",
+    "selectSkinTone": "انتخاب رنگ پوست",
+    "remove": "حذف ایموجی",
+    "categories": {
+      "smileys": "لبخندی‌ها",
+      "people": "آدمک‌ها",
+      "animals": "حیوانات و طبیعت",
+      "food": "غذا و نوشیدنی",
+      "activities": "فعالیت‌ها",
+      "places": "مسافرت",
+      "objects": "اشیا",
+      "symbols": "نماد‌ها",
+      "flags": "پرچم‌ها",
+      "nature": "طبیعت",
+      "frequentlyUsed": "استفاده‌شده"
+    }
+  }
+}

